test/cbmc/proofs/ARP/ARPRefreshCacheEntry/ARPRefreshCacheEntry_harness.c

```diff
@@ -11,6 +11,19 @@ void harness()
     MACAddress_t xMACAddress;
     uint32_t ulIPAddress;
 
-    vARPRefreshCacheEntry( &xMACAddress, ulIPAddress );
-    vARPRefreshCacheEntry( NULL, ulIPAddress );
+    /*
+    For this proof, its assumed that the endpoints and interfaces are correctly
+    initialised and the pointers are set correctly.
+    Assumes one endpoints and interface is present.
+    */
+
+    pxNetworkEndPoints = ( NetworkEndPoint_t * ) malloc( sizeof( NetworkEndPoint_t ) );
+    __CPROVER_assume( pxNetworkEndPoints != NULL );
+    __CPROVER_assume( pxNetworkEndPoints->pxNext == NULL );
```

Comment:
Why not try this with more than one network endpoint?

Reply:
updated